scenarios/reproducers/bgp-4-racks-3-ocps.yml

```diff
@@ -0,0 +1,402 @@
+        <bridge name='l31-s0' stp='on' delay='0'/>
+      </network>
+    l31-s1: |
+      <network>
+        <name>l31-s1</name>
+        <bridge name='l31-s1' stp='on' delay='0'/>
+      </network>
+    # leafs to nodes
+    ## rack0
+    l00-node0: |
+      <network>
+        <name>l00-node0</name>
+        <bridge name='l00-node0' stp='on' delay='0'/>
+      </network>
+    l01-node0: |
+      <network>
+        <name>l01-node0</name>
+        <bridge name='l01-node0' stp='on' delay='0'/>
+      </network>
+    ## rack1
+    l10-node0: |
+      <network>
+        <name>l10-node0</name>
+        <bridge name='l10-node0' stp='on' delay='0'/>
+      </network>
+    l11-node0: |
+      <network>
+        <name>l11-node0</name>
+        <bridge name='l11-node0' stp='on' delay='0'/>
+      </network>
+    ## rack2
+    l20-node0: |
+      <network>
+        <name>l20-node0</name>
+        <bridge name='l20-node0' stp='on' delay='0'/>
+      </network>
+    l21-node0: |
+      <network>
+        <name>l21-node0</name>
+        <bridge name='l21-node0' stp='on' delay='0'/>
+      </network>
+    # leafs to ocp
+    ## rack3
+    l30-ocp0: |
+      <network>
+        <name>l30-ocp0</name>
+        <bridge name='l30-ocp0' stp='on' delay='0'/>
+      </network>
+    l30-ocp1: |
+      <network>
+        <name>l30-ocp1</name>
+        <bridge name='l30-ocp1' stp='on' delay='0'/>
+      </network>
+    l30-ocp2: |
+      <network>
+        <name>l30-ocp2</name>
+        <bridge name='l30-ocp2' stp='on' delay='0'/>
+      </network>
+    l31-ocp0: |
+      <network>
+        <name>l31-ocp0</name>
+        <bridge name='l31-ocp0' stp='on' delay='0'/>
+      </network>
+    l31-ocp1: |
+      <network>
+        <name>l31-ocp1</name>
+        <bridge name='l31-ocp1' stp='on' delay='0'/>
+      </network>
+    l31-ocp2: |
+      <network>
+        <name>l31-ocp2</name>
+        <bridge name='l31-ocp2' stp='on' delay='0'/>
+      </network>
+  vms:
+    controller:
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
+        }}
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - ocpbm
+        - osp_trunk
+    compute:
+      amount: "{{ cifmw_libvirt_manager_compute_amount }}"
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
+        }}
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "centos-stream-9.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      spineleafnets:
```

Comment:
Why are we not using different "roles"
i.e there is a vm of type ocp-1 with amount = 1, a vm of type ocp-2 also amoutn = 1

```
vms:
  ocp-1:
    amount: 1
     xml_paths:
        - /home/dev-scripts/ocp_master_0.xml
    networks:
       - ocpbm
       - l10-ocp0
       - l20-ocp0
       - l30-ocp0
  ocp-1:
    networks:
       - ocpbm
       - l10-ocp1
       - l20-ocp1
       - l30-ocp1
    amount: 1
     xml_paths:
        - /home/dev-scripts/ocp_master_1.xml
  ocp-2:
    amount: 1
     xml_paths:
        - /home/dev-scripts/ocp_master_2.xml
    networks:
       - ocpbm
       - l10-ocp3
       - l20-ocp3
       - l30-ocp3
```

Reply:
If we need a group "ocp" and a group "compute" in one of these ansible inventories - we could add something to tell which group vms should be in.

```
vms:
  ocp-1:
     amount: 1
     ansible_group: ocp
  ocp-2:
     amount: 1
     ansible_group: ocp
  ocp-3:
     amount: 1
     ansible_group: ocp
```

Reply:
Thanks for the proposal. If I have time, I will try to apply this change on a later PR.